### ISSUE TITLE
how to make this plugin ros2 agnostic?

### ISSUE THREAD

**Issue by xbkaishui** [tesseract-robotics/tesseract_trac_ik#1]
Hello: 
 how to make this plugin ros2 agnostic, I want to use this plugin in windows. 

Thanks

**Comment by xbkaishui** [comment]
hello， any update on this

**Comment by Levi-Armstrong** [comment]
This package is wrapper around `trac_ik_lib` which does depend on ament build tool but that is outside of our control. It would be easy to update `trac_ik_lib` to avoid using ament, but not sure if the maintainers accept this change.

**Comment by Levi-Armstrong** [comment]
I posted the question [here](https://github.com/aprotyas/trac_ik/issues/35).

**Comment by xbkaishui** [comment]
OK， can we conside this repo [tracikpy](https://github.com/mjd3/tracikpy/tree/main), this is no depends on ros2

**Comment by xbkaishui** [comment]
there is another issue, I saw trac_ik is not easy to install  on windows
